src/game.c

```diff
@@ -457,6 +487,20 @@ int game_event(Game *game, const SDL_Event *event)
         return 0;
     } break;
 
+    case GAME_STATE_CREDITS: {
+        switch (event->type) {
+        case SDL_KEYDOWN: {
+            if (event->key.keysym.sym == SDLK_ESCAPE) {
+                game_switch_state(game, GAME_STATE_LEVEL_PICKER);
+                return 0;
+            }
+        } break;
+        }
+
+        credits_event(&game->credits, &game->camera, event);
```

Comment:
`&game->credits` this is why I had null pointer, when calling credits->list_selector, but centering still doesn't work. It seems that I have to begin from scratch.